src/ai/ai.ts

```diff
@@ -0,0 +1,172 @@
+import _ from 'lodash';
+import Service from '@ainize-team/ainize-js/dist/service';
+
+import FactoryBase from '../factoryBase';
+import AinftObject from '../ainft721Object';
+import { OperationType, getService, request } from '../ainize';
+import {
+  AiConfigurationTransactionResult,
+  CreditTransactionResult,
+  NftToken,
+  NftTokens,
+  QueryParams,
+  TokenStatus,
+} from '../types';
+import { buildSetTxBody, buildSetValueOp, sleep, sendTx } from '../utils/util';
+import { Path } from '../utils/path';
+import { validateObject, validateObjectOwner } from '../utils/validator';
+import { DEFAULT_AINIZE_SERVICE_NAME } from '../constants';
+
+/**
+ * This class manages ai configurations for AINFT object,\
+ * manages the credits needed for their uses, and checks if tokens are available for creation.\
+ * Do not create it directly; Get it from AinftJs instance.
+ */
+export class Ai extends FactoryBase {
```

Comment:
minor) if this class is used for chatbot usecases only (e.g. 'ImageGen' class for image generation AI), you can consider 'Persona' as a name, but it's basically up to you. :)

Reply:
it's a good idea, but i think this class is used in general ai use cases (e.g. configuration, credit, utils)